__tests__/retrieve_date__test.js

```diff
@@ -3,231 +3,305 @@ const sinon = require('sinon');
 
 const datetimeUtils = require('../src/main').default;
 
-describe('should retrieve period string', () => {
+describe('#retrievePeriod', () => {
   describe('without baseDate', () => {
-    let clock;
-    beforeAll(() => {
-      clock = sinon.useFakeTimers(new Date('2017-03-20').getTime());
-    });
+    describe('when current date is 20/03/2017', () => {
+      let clock;
+      beforeAll(() => {
+        clock = sinon.useFakeTimers(new Date('2017-03-20').getTime());
+      });
 
-    it('should retrieve today', () => {
-      const period = datetimeUtils.retrievePeriod('today');
-      expect(period.start).to.equal('2017-03-20');
-      expect(period.end).to.equal('2017-03-20');
-    });
+      it('should today period be 19/03/2017 (interpreted as yesterday, as today is deprecated)', () => {
```

Comment:
This description doesn't make much sense :thinking:

Reply:
Same for the ones below actually. I can help if neeeded

Reply:
Also I don't think you should hardcode outputs in the `it` sentence, nor should you comment on deprecations there

Reply:
I'll remove deprecation warning, but how would you sentence it? 

IMO putting expected value in the `it` statement is ok as it would help to validate behavior just by doing `yarn test --verbose`

Reply:
<img width="900" alt="screenshot 2018-11-29 at 5 00 41 pm" src="https://user-images.githubusercontent.com/4658821/49210550-56e3aa80-f3f8-11e8-8348-7d49d5500076.png">

Reply:
IMO the text should be explicit enough you don't have to put the expected output in it :) Plus it's the kind of thing that's inevitably gonna get out of sync over time